src/components/input-field/link-helper.ts

```diff
@@ -0,0 +1,70 @@
+export function getHref(value) {
+    const href = value ? String(value.trim()) : '';
+    if (isValid(href)) {
+        return href;
+    }
+
+    return prependProtocol(href);
+}
+
+function isValid(href) {
+    return (
+        hasKnownProtocol(href) ||
+        isRelativeLink(href) ||
+        hasRelativeProtocol(href) ||
+        hasMailtoProtocol(href)
```

Comment:
Since we have a separate type for email addresses, I'm thinking we probably don't want to allow them here.

Reply:
:zap: